lib/plugins/chat.js

```diff
@@ -165,7 +165,7 @@ function inject (bot, options) {
     })
   }
 
-  async function tabComplete (text, assumeCommand = false, sendBlockInSight = true) {
+  async function tabComplete (text, assumeCommand = false, sendBlockInSight = true, timeout = 100) {
```

Comment:
100ms seems rather low of a timeout for high latency connections

Reply:
Then someone can change it for their use case. But if you think that e.g. 200 would be better, then I can change it

Reply:
How about now? It's based on bot's ping

Reply:
No, that's worse. Ideally we stick to something at least 1 second, high latency connections are perfectly valid, they should not be an error case. Also, we shouldn't have every function having inconsistent timeout values, that makes things very messy.

I just checked the mineflayer code for existing timeouts, for blocks.js and command blocks and uses [5000ms timeout](https://github.com/PrismarineJS/mineflayer/blob/master/lib/plugins/place_block.js#L14), so I'd say use that here also.

Reply:
I don't see why dynamic timeout would be worse, but alright, I will change it to 5s